### ISSUE TITLE
VanBUG talking points

### ISSUE THREAD

**Issue by sjackman** [hackseq/October_2016#18]
Do you have any suggested talking points for me for VanBUG this Thursday?

**Comment by sjackman** [comment]
Amy wrote…

> Hi VanBUG Dev Team, 
> We are organizing a genomics hackathon in Oct (#hackseq, http://www.hackseq.com/). Would it be okay for us to advertise and make an announcement this Thursday at VanBUG? 
> Shaun would talk a bit about the overall plan/organization for the hackathon, as well as making a call for applicants to be team leaders and participants. It should take about 5 -10 minutes. 
> Also, the #hackseq team is interested in finding out whether VanBUG is interested in being a community partner. It could be a great opportunity to advertise for VanBUG at the hackathon as well. 
> Cheers, 
> Amy

**Comment by santina** [comment]
Here are some that came to mind: 
- Give an idea of what HackSEQ is like. This includes some examples projects of appropriate length/depth for the limited time frame, the skill sets we're looking for, and how they will be judged,  who are judging (so to bring up some familiar names of PIs) 
- Application process (i.e. team leaders first now and then applicants choose the projects) 
- Various types of involvements (mentors, team leaders, participants, etc) 
- Resources that are available to participants (AWS credits, wifi, mentorship, travel grant, other sponsorship, etc) 
- What immediate actions they can take now (checking out the website, signing up for the mailing list, applying to be a team leader, start thinking about an applicable project, etc) 
- Ask if anyone has questions about what you just said  

Basically make it sounds like all the cool kids and your friends are gonna be in it :)

**Comment by sjackman** [comment]
When are the project proposals due?

**Comment by sjackman** [comment]
Here's an example script. This won't be memorized, but gives you an idea of what I plan to say.

The ASHG conference, American Society of Human Genetics, is being held in Vancouver this year in mid-October. The weekend before the conference we're organizing a bioinformatics hackathon, HackSeq, taking place Saturday to Monday, October 15th to 17th. The hackathon works like so. People interested in organizing and leading a team of participants submit a brief project proposal. These project descriptions are distributed to all the participants, and the participants rank the projects that they would like to work on. We match up projects with teams of participants based on their rankings, and those projects with enough participants will go ahead. We provide resources to enable the projects, including high-performance computing with Amazon Web Services Elastic Cloud Computing, Internet access, mentorship, travel grants and food. Please check out our web site at Hackseq.com if you're interested in either submitting a project proposal or participating in the hackathon. You can sign up to the mailing list to get more details and reminders as they come out. Project proposals are due June 2nd. It'll be a lot of fun, and I hope that you'll join us!

**Comment by ababaian** [comment]
I think we were saying April 30th as the due date but the abstract deadline for ASHG is now June 2, 2016.

**Comment by sjackman** [comment]
Should we match our proposal deadline up with ASHG?

**Comment by ttimbers** [comment]
@sjackman, sure, I think it makes sense to match-up our proposal deadline with ASHG. How about Team Leader apps are due June 2nd. Then we can open participant applications (or get people who already registered to rank the team leader submitted projects) and have them still submitted by June 30th. @ababaian @santina @adrisua what do you all think?

**Comment by ttimbers** [comment]
and @sjackman your example script looks great!

**Comment by sjackman** [comment]
June 2nd sounds good to me. Thanks, Tiffany!

**Comment by ababaian** [comment]
If I remember, originally we wanted the *participant* applications to be due ~ June 2nd so they could book their plane tickets accordingly. That might not work if we added an additional team leader's choosing their teams to the process.

Unless we do something like
- April 30th Team Leader Due.
- May 15 Participant Due.
- June 1st Decide on teams.

 Not sure if that's feasible tbh.
